### ISSUE TITLE
Webtrees 2.2.0 version released but not available here

### ISSUE THREAD

**Issue by Neriderc** [NathanVaughn/webtrees-docker#154]
The latest version 2.2.0 has been [released](https://github.com/fisharebest/webtrees/releases/tag/2.2.0) a couple of days ago, but this container hasn't been updated.

To be fair, [this one](https://github.com/H2CK/webtrees) hasn't either, and webtrees hasn't told me about the update.

Is there some version check not working for this new version?

P.S. thanks for providing this container, I've been a happy customer for many years :slightly_smiling_face:

**Comment by NathanVaughn** [comment]
You would be correct, it bombed out since I didn't have a PHP version defined for 2.2: https://github.com/NathanVaughn/webtrees-docker/actions/runs/11995958443

I added a new entry for PHP 8.4 with webtrees 2.2 today, but that also failed due to Imagick I think building for ARM: https://github.com/NathanVaughn/webtrees-docker/actions/runs/11997681382

I downgraded it to 8.3 and was just going to let it try again tomorrow since I was running out of patience.

**Comment by Neriderc** [comment]
Haha all good, I just wanted to make sure you were aware. Fix it up when it suits you 🙂

Cheers

**Comment by NathanVaughn** [comment]
Version 8.3 failed today too. Ugh. Possibly related to Imagick/imagick#698. I was able to make an x86 build on my machine, I'll have to play around with ARM with QEMU to debug. May have to disable Imagick for ARM to get 2.2.0 out at some point.

**Comment by Neriderc** [comment]
Yeah I've had a play too, I wasn't as successful as you, couldn't even get it to build Imagick locally.

Possibly related (but not helpful): https://github.com/Imagick/imagick/issues/697

I'd happily accept a x86 version for my dev environment, so I can test some fixes for a module I maintain. Apparently all the modules that use app() are broken in webtrees 2.2.0 so my key reason for wanting the update so early is to be able to resolve this.

Or if you have an updated Dockerfile I can build myself, I'm not that familiar with Dockerfiles and haven't managed to get it working with PHP8.3 and webtrees 2.2.20/.

**Comment by NathanVaughn** [comment]
With the repo cloned, I ran `uv run python .\dev\build_test_version.py --webtrees_version 2.2.0` which fires off a build of Webtrees 2.2.0 with the appropriate PHP versions and everything passed as build arguments to Docker.

**Comment by Neriderc** [comment]
That fails for me, the same as running the dockerfile. 

```
ERROR: failed to solve: process "/bin/sh -c pecl install imagick  && docker-php-ext-enable imagick  && docker-php-ext-configure gd --with-freetype --with-jpeg  && docker-php-ext-install -j\"$(nproc)\" pdo pdo_mysql pdo_pgsql zip intl gd exif" did not complete successfully: exit code: 1
```

It's the imagick stuff. You didn't make other changes?

**Comment by NathanVaughn** [comment]
Uh, no. I cleared my Docker build cache and tried again and it still worked. I'm not sure. 
![image](https://github.com/user-attachments/assets/daadae03-5736-4bec-8e2b-de997b96cdf7)

I ran the above command on commit b274df3 with no local changes.

**Comment by Neriderc** [comment]
All good. I have now managed to build without Imagick and I think I can probably do a good enough job without it.

Thanks for the help, I'll let you sort the pre-built containers, I'm certainly out of my depth and would be no help :slightly_smiling_face:

**Comment by NathanVaughn** [comment]
I'm able to replicate locally. The issue I get is https://github.com/Imagick/imagick/issues/640.

**Comment by Neriderc** [comment]
Yes, same issue I was getting. Looks like it's fixed but not released, and a comment at the bottom has a workaround until it gets into a release. Hopefully it works!

**Comment by NathanVaughn** [comment]
I think I got it situated. 2.2.0 tags have been pushed. I did a quick test locally and it seems to work. Let me know if it looks good.

**Comment by Neriderc** [comment]
It's working for me, well done! I am using x86 though, haven't tested ARM.
